src/middleware/mockAuth.ts

```diff
@@ -0,0 +1,10 @@
+import type { MiddlewareHandler } from "hono";
+
+export const mockAuth: MiddlewareHandler = async (c, next) => {
+  c.set("user", {
+    id: "user-123",
+    role: "editor",
+    email: "camilo@tuna.com",
+  });
+  await next();
+};
```

Comment:
is this being used? 

also would go for kebab case file and folder name structure but again [nitpick]